tests/app/forms/field_handlers/test_text_area_handler.py

```diff
@@ -73,3 +73,24 @@ def test_get_length_validator_with_max_length_override():
     validator = text_area_handler.get_length_validator()
 
     assert validator.max == 30
+
+
+def test_get_textarea_row_length():
+    answer = {
+        "guidance": "",
+        "id": "answer",
+        "rows": 3,
+        "label": "Enter your comments",
+        "mandatory": False,
+        "q_code": "0",
+        "type": "TextArea",
+    }
+
+    text_area_handler = TextAreaHandler(answer, {}, AnswerStore(), {})
```

Comment:
The last three args are defaulted if not passed, so this can be simplified to `TextAreaHandler(answer)`.